startleft/startleft/log.py

```diff
@@ -42,3 +48,28 @@ def get_log_level(ctx, param, value):
         return levels[value.upper()]
     else:
         raise click.BadParameter(f"Log level must be one of: {', '.join(levels.keys())}")
+
+
+def get_root_log():
+    return logging.getLogger()
+
+
+def get_uvicorn_error_log():
```

Comment:
I will change as well the method name to **get_uvicorn_logger** as it is returning the uvicorn Logger and removing the word _error_ because although the Logger's name is _uvicorn.error_, at a first glance, can lead to thinking that you are getting only the error Logger.

Reply:
I agree. The change has been done